diff --git a/static/ngx_http_brotli_static_module.c b/static/ngx_http_brotli_static_module.c
--- a/static/ngx_http_brotli_static_module.c
+++ b/static/ngx_http_brotli_static_module.c
@@ -267,3 +267,4 @@
   ngx_str_set(&content_encoding_entry->key, kContentEncoding);
   ngx_str_set(&content_encoding_entry->value, kEncoding);
   req->headers_out.content_encoding = content_encoding_entry;
+  req->allow_ranges = 1;
